src/screens/SettingsScreen/SettingsScreen.test.tsx

```diff
@@ -10,6 +11,14 @@ jest.mock('react-redux', () => {
   return {
     __esModule: true,
     useDispatch: jest.fn().mockImplementation(() => jest.fn()),
+    useSelector: jest.fn().mockReturnValue({
+      username: 'dummy-username',
+      userImage: 'dummy-image-url',
+    }),
+    useNavigation: () =>
```

Comment:
You should create a separate mock for `useNavigation` (I demonstrated it in my example). But on the other hand we should consider if we need it, as I mentioned in my previous comment :smile:

Reply:
And in this case it won't be needed, but have checked your example branch, thanks for advice!